unrealengine.js

```diff
@@ -120,24 +120,26 @@ try {
     } else {
       await p.locator('.btn .add').click();
       console.log('  ↳ Added to cart');
+      await page.waitForTimeout(1000);
     }
     ids.push(id);
   }
   if (!ids.length) {
     console.log('Nothing to claim');
   } else {
+    await page.waitForTimeout(1000);
+    console.log('Click shopping cart');
+    await page.locator('.shopping-cart').click();
     await page.waitForTimeout(2000);
     const price = (await page.locator('.shopping-cart .total .price').innerText()).split(' ');
     console.log('Price: ', price[1], 'instead of', price[0]);
-    if (price[1] != '0') {
+    if (price[1] != '0' && price[1] != '$0.00') {
       const err = 'Price is not 0! Exit! Please <a href="https://github.com/vogler/free-games-claimer/issues/44">report</a>.';
```

Comment:
I wonder if we should maybe print the price here, just so that people see what it is instead and thus can report better issues in the future?

```suggestion
      const err = `Price is ${price[1]} and thus not 0! Exit! Please <a href="https://github.com/vogler/free-games-claimer/issues/44">report</a>.`;
```